NewForce-Cohort-9/tabloidmvc-nacua-matata
# Ks branch1

Description:
Completed trello ticket #13 view all tags https://trello.com/c/fSluAWDN. This PR adds view all tags functionality for admins.

Type of Change: New Feature

Testing Instructions:

1. git add and git commit whatever you're working on on your branch
2. git checkout main
3. git fetch --a
4. git checkout ks-branch1
5. If you don't see my most recent changes, then git pull origin ks-branch1
6. Create the Tabloid MVC database if you haven't done so already
7. Click the solid green triangle button to execute with debugger
8. login as admin with email: admin@example.com - you will be directed to the home page
9. You should see the home page with a navigation bar at the top
10. Click the option 'Tag Management'. You will be directed to a new page that lists all of the tags in alphabetical order.
11. It should look like this: C#, Cyclopean Terrors, Family, JavaScript.
12. log out using the 'log out' option on the navbar
13. you should see a message informing you that you have successfully logged out of the application
14. close out the web page 

Checklist:
 My code follows the style guidelines of this project
 I have performed a self-review of my code
 I have commented my code, particularly in hard-to-understand areas
 My changes generate no new warnings or errors
 I have added test instructions that prove my fix is effective or that my feature works
